### PR TITLE
Sort call feeds consistently when choosing active speaker

### PR DESCRIPTION
**Pull request by dbkr**
As hopefully explained by comment

For https://github.com/vector-im/element-call/issues/331

### CODE REVIEW

File: src/room/usePTT.ts [hunk 1]
```diff
@@ -30,6 +30,17 @@ function getActiveSpeakerFeed(
...
+  // make sure the feeds are in a deterministic order so every client picks
+  // the same one as the active speaker
+  const collator = new Intl.Collator("en", {
+    sensitivity: "variant",
+    usage: "sort",
+    ignorePunctuation: false,
+  });
+  activeSpeakerFeeds.sort((a: CallFeed, b: CallFeed): number =>
+    collator.compare(a.userId, b.userId)
+  );
```
Comment (robintown) [inline src/room/usePTT.ts:42]: Why isn't `activeSpeakerFeeds.sort()` deterministic on its own?

**Review by robintown — COMMENTED** [review]
a question

File: src/room/usePTT.ts (hunk 1 above)
Comment (robintown) [reply src/room/usePTT.ts:42]: Nevermind, was blind to the fact that we're sorting by the `userId` key

**Review by robintown — APPROVED** [review]

File: src/room/usePTT.ts (hunk 1 above)
Comment (dbkr) [reply src/room/usePTT.ts:42]: To be fair, this could be better commented - I'll add one.

**Pull request merged**